libr/util/graph.c

```diff
@@ -312,3 +312,121 @@ R_API void r_graph_dfs(RGraph *g, RGraphVisitor *vis) {
 		free (color);
 	}
 }
+
+typedef struct _dfs_inserter {
+	RGraph *g;
+	HtUP *reverse;	//reverse lookup of nodes
+//	RList *mo;	//multiple out nodes
+	RList *mi;	//multiple in nodes
```

Comment:
@trufae at some point these should be turned into Vec's for better memory locality / performance. Also for RGraph in general.

Reply:
tbh i think a stack or queue might be better here, but list is more flexible